sys/src/9/amd64/kernel.ld

```diff
@@ -1,78 +1,59 @@
-/* Simple linker script for the ROS kernel.
-   See the GNU ld 'info' manual ("info ld") to learn the syntax. */
-
-/* This script needs to be invoked with -z max-page-size=0x1000.  Otherwise,
- * ld will offset our first section to 1MB within the actual file.  Multiboot
- * requires the header to be in the first two pages. */
+/*
+ * Linker script for Harvey.
+ */
 
-OUTPUT_FORMAT("elf64-x86-64", "elf64-x86-64", "elf64-x86-64")
 ENTRY(_start)
-/* start the kernel at 0x110000. 
- * That way we can use lower ram for critical structures
- */
-KERN_LOAD_ADDR = 0xfffffffff0000000;
 
-SECTIONS
-{
-	/* Entry Linked and loaded at 0x00100000 (includes multiboot) */
-	. = 0x00110000;
+SECTIONS {
+	/*
+	 * start the kernel at 0x0xffff_8000_0020_0000
+	 * This preserves some RAM between 1MiB and the
+	 * start of the kernel for critical structures.
+	 */
+	. = 0xffff800000100000;
 
-	.bootstrap : {
+	PROVIDE(boottext = .);
+	.text.boot : ALIGN(4096) {
 		*(.boottext .bootdata)
+		. = ALIGN(4096);
+		PROVIDE(eboottext = .);
```

Comment:
are eboottext and esys used anywhere?  I see no reference to them

Reply:
No, same thing as `text`: they appear in a symbol dump for debugging.